src/map/battle.c

```diff
@@ -4716,7 +4710,22 @@ static struct Damage battle_calc_weapon_attack(struct block_list *src, struct bl
 			flag.cri = 1;
 	}
 	if (flag.cri) {
+#ifdef RENEWAL
+		if (battle_config.feature_enable_multi_crit == true) { // Turn Multi-Critical ON/OFF in config [X-EcutiOnner]
+	#if PACKETVER >= 20161207
+			if ((wd.type & BDT_MULTIHIT) != 0)
+				wd.type = BDT_MULTICRIT;
+			else
+				wd.type = BDT_CRIT;
+	#else
+			wd.type = BDT_CRIT;
+	#endif
+		} else {
+			wd.type = BDT_CRIT;
+		}
+#else
 		wd.type = BDT_CRIT;
+#endif
```

Comment:
Since you always set `wd.type` to `BDT_CRIT` except for one special case, you could replace your if-else-jungle with this:
```
		wd.type = BDT_CRIT;
#ifdef RENEWAL
#if PACKETVER >= 20161207
		if (battle_config.feature_enable_multi_crit == 1 && (wd.type & BDT_MULTIHIT) != 0)
			wd.type = BDT_MULTICRIT;
#endif
#endif
```